stdlib/2and3/ftplib.pyi

```diff
@@ -0,0 +1,123 @@
+# Stubs for ftplib (Python 2.7/3)
+import sys
+from typing import Optional, BinaryIO, Tuple, TextIO, Iterable, Callable, List, Union, Iterator, Dict, Text, TypeVar, Generic, AnyStr
+from types import TracebackType
+from socket import socket
+from ssl import SSLContext
+
+MSG_OOB = ...  # type: int
+FTP_PORT = ...  # type: int
+MAXLINE = ...  # type: int
+CRLF = ...  # type: str
+if sys.version_info >= (3,):
+    B_CRLF = ...  # type: bytes
+
+class Error(Exception): ...
+class error_reply(Error): ...
+class error_temp(Error): ...
+class error_perm(Error): ...
+class error_proto(Error): ...
+
```

Comment:
Missing `all_errors`, which is a tuple of exception classes (https://docs.python.org/3/library/ftplib.html#ftplib.all_errors).